src/components/dialogs/BuyTokensDialog.jsx

```diff
@@ -0,0 +1,222 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+// import { I18n } from 'react-redux-i18n'
+import { CSSTransitionGroup } from 'react-transition-group'
+
+
+import { RaisedButton, TextField } from 'material-ui'
+
+import ModalDialog from './ModalDialog'
+import RateHistoryChart from '../exchange/RateHistoryChart'
+
+import { modalsOpen, modalsClose } from 'redux/modals/actions'
+
+import './BuyTokensDialog.scss'
+
+export class BuyTokensDialog extends React.Component {
+
+  static propTypes = {
+    order: PropTypes.object,
+    handleClose: PropTypes.func
+  }
+
+  render() {
+
+    let icons = {
+      time: require('assets/img/icn-time.svg'),
+      ethereum: require('assets/img/icn-ethereum.svg'),
+    }
+
+    return (
+      <CSSTransitionGroup
+        transitionName="transition-opacity"
+        transitionAppear={true}
+        transitionAppearTimeout={250}
+        transitionEnterTimeout={250}
+        transitionLeaveTimeout={250}>
+        <ModalDialog onClose={() => this.props.handleClose()}>
+          <div styleName="root">
+            <div styleName="header">
+              <div styleName="row">
+                <div styleName="col1">
+                  <h3>Buy TIME Tokens</h3>
+                  <div styleName="balance">
+                    <div styleName="label">Balance:</div>
+                    <div styleName="value">
+                      <span styleName="value1">1 512 000</span>
+                      <span styleName="value2">.00124 ETH</span>
+                    </div>
+                  </div>
+                </div>
+                <div styleName="col2">
+                  <div className="ByTokensDialog__icons">
+                    <div className="row">
+                      <div className="col-xs-1">
+                        <div className="icon">
+                          <div className="content" style={{
+                            background: `#05326a url(${icons.time}) no-repeat center center`
```

Comment:
No
1) We will remove this code after we got an ability to upload icons and specify colors;
2) Icons will not be embedded, they will be provided (loaded), so there will be an image as object, not as a style.

Won't wix